src/main/java/baseball/global/util/Validator.java

```diff
@@ -0,0 +1,53 @@
+package baseball.global.util;
+
+import static baseball.global.constant.BaseballConstant.PLAY_AMOUNT;
+import static baseball.global.constant.BaseballConstant.SIGN_RESTART;
+import static baseball.global.constant.BaseballConstant.SIGN_STOP;
+import static baseball.global.constant.ErrorMessage.NUMBER_DUPLICATION_ERROR_MESSAGE;
+import static baseball.global.constant.ErrorMessage.NUMBER_FORMAT_ERROR_MESSAGE;
+import static baseball.global.constant.ErrorMessage.NUMBER_LENGTH_ERROR_MESSAGE;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+public class Validator {
+
+    public static void validatePlayerNumber(String[] inputNumbers) {
+        Validator.validateNumbersFormat(inputNumbers);
+        Validator.validateNumbersLength(inputNumbers);
+        Validator.validateNumbersDuplication(inputNumbers);
+    }
+
+    private static void validateNumbersLength(String[] numbers) {
+        if (numbers.length != PLAY_AMOUNT) {
+            throw new IllegalArgumentException(NUMBER_LENGTH_ERROR_MESSAGE);
+        }
+    }
+
+    private static void validateNumbersDuplication(String[] numbers) {
+        int noneDuplicateNumber = Arrays.stream(numbers).collect(Collectors.toSet()).size();
```

Comment:
```suggestion
        int noneDuplicateNumber = Arrays.stream(numbers)
             .collect(Collectors.toSet())
             .size();
```

물론 스트림의 경우 한 줄로 간략하게 쓸 수도 있겠지만, stream() 메서드 이후에는 도트마다 개행을 하는 것이 가독성이 좋아 많이 사용되는 추세입니다! 

또, noneDuplicatedNumber 를 구하는 로직 자체를 메서드로 분리해볼 수도 있을 것 같은데 승준님은 어떻게 생각하시는지 궁금합니다 😆